src/tests/SharpRaven.UnitTests/Data/JsonPacketFactoryTests.cs

```diff
@@ -115,7 +115,9 @@ public void Create_Project_EventIDIsValidGuid()
             var json = this.jsonPacketFactory.Create(project, (SentryMessage)null);
 
             Assert.That(json.EventID, Is.Not.Null.Or.Empty, "EventID");
+#if (!net35)
             Assert.That(Guid.Parse(json.EventID), Is.Not.Null);
+#endif
```

Comment:
What's not .NET 3.5 compatible about this? Can we change something to make it compatible with all target frameworks?